src/components/Header.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import {Navbar, NavbarBrand} from 'reactstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
Since you're using bootstrap everwhere can you please find a way to import it automatically everywhere.

Reply:
Okay, will try this